content/swiftui/concepts/Lists.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: "SwiftUI List" 
+Description: "Building Lists in SwiftUI"
+Subjects:  
+  - "iOS Development"
+Tags:
+  - "SwiftUI" 
+  - "iOS"
+CatalogContent: 
+  - "learn-swift"
+  - "paths/ios-developer"
+---
+
+# SwiftUI Lists
+
+Lists in **SwiftUI** are single-column containers that vertically display a collection of items. The `List` component is used to create this feature. 
+
+## How to Create a List
+
+To set up your list, follow these steps:
+
+1. Click on **File**, then click on **New**.
+2. Click on **Multiplatform** and **Swift Package**.
+3. Give your folder a label, then click **Create**. You will see the standard project from there.
```

Comment:
This is more a tutorial style and not so much reference style (Docs styling)

Reply:
Do you have any suggestions on how to make it more reference-like @yangc95? I struggled with that a little bit, so I could use your help. :)

Reply:
Yeah, so for these concepts, they don't need to go over step-by-step in a tutorial fashion of how to get something set up. To create a list, we're going to go right into the technicalities of using the `List` structure and show that you can add views into it. You can use a pseudo code block to display this like:


```pseudo
List {
    \\ Views here
}
```

I suggest nesting text views as the list element